packages/components/src/pages/ChatSearch.vue

```diff
@@ -4,6 +4,10 @@
     @mousemove="makeResizing"
     @mouseup="stopResizing"
     @mouseleave="stopResizing"
+    @drop="onDrop"
+    @dragover.prevent
+    @dragevent.prevent
```

Comment:
The event handler `dragevent` is not a standard DOM event. The correct event for handling drag operations is `dragover`, which is already present.

\- Navie